source/distributed/solution_transfer.cc

```diff
@@ -404,6 +404,14 @@ namespace parallel
           unpack_dof_values<typename VectorType::value_type>(data_range,
                                                              dofs_per_cell);
 
+      if (std::accumulate(dof_values.begin(),
+                          dof_values.end(),
+                          0,
+                          [](const auto &a, const auto &b) {
+                            return a + b.size();
+                          }) == 0)
+        return; // no data to interpolate if all sizes equal to zero
```

Comment:
@marcfehling Could you check here if we need to clear the dof values here.

Reply:
I don't think we need to. If `data_range` is empty, `dof_values` will stay empty, too, and would be destroyed anyways as it's a local container.

I think that checking if `data_range` is empty would achieve the same as the `std::accumulate` construct, or am I wrong?